src/components/SessionApiLogout.tsx

```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+
+import * as React from 'react';
+
+export const SessionApiLogout = async () => {
```

Comment:
nits: PascalCase is used as component's name. This is just a function, so this file and exported func should be renamed to camelCase